lib/tasks/republish_editions_from_files.rake

```diff
@@ -0,0 +1,153 @@
+# Enqueue republishing for editions listed in a text file
+#
+# Summary
+# - Reads a list of lines and extracts document IDs.
+# - The task deduplicates IDs and (optionally) limits to a TEST_BATCH for safe trials.
+# - Enqueueing uses the bulk republishing pattern:
+#   `PublishingApiDocumentRepublishingWorker.perform_async_in_queue("bulk_republishing", doc_id, true)`
+#
+# Usage with kubernetes:
+#   k exec -it deploy/whitehall-admin -- rake 'data_migration:enqueue_republishing_from_file'
+#
+# Usage:
+#    govuk-docker-run bundle exec rake "data_migration:enqueue_republishing_from_file"
+
+namespace :data_migration do
+  desc "Enqueue republishing for editions listed in text/log file"
+  task enqueue_republishing_from_file: :environment do
+    require "thor"
+    shell = Thor::Shell::Basic.new
+    input_file = (ENV["FILE"] || "further_filtered_documents.txt").to_s.strip
```

Comment:
We should use a rake task argument rather than look for an ENV variable here. Makes it easier to call, and less likely some external process could mess with it.